blog/_posts/2021-06-01-May-2021-update.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "May 2021 Update"
+description: ""
+tags: []
+---
+{% include JB/setup %}
+
+Dear GeoCAT Community,
+
+Our team would like to share May, 2021 update 
+about GeoCAT releases and community activities below:  
+
+### - GeoCAT-comp v2021.05.0 and v2021.05.1 Released!
+With these release, new functions such as Fourier smoothing, atmospheric 

```

Comment:
```suggestion
With these releases, new functions such as Fourier smoothing, atmospheric 

```
I had a different edit before, but if these are two separate releases, it should be "with these releases" as noted. If it is only one release it would be the prior comment of "with this release"